actpool/actpool.go

```diff
@@ -202,7 +202,7 @@ func (ap *actPool) Add(ctx context.Context, act action.SealedEnvelope) error {
 	}
 	if ap.gasInPool+intrinsicGas > ap.cfg.MaxGasLimitPerPool {
 		actpoolMtc.WithLabelValues("overMaxGasLimitPerPool").Inc()
-		return errors.Wrap(action.ErrActPool, "insufficient gas space for action")
+		return errors.Wrap(action.ErrHitGasLimit, "insufficient gas space for action")
```

Comment:
change `ErrHitGasLimit` to `errors.New("insufficient gas space for action")`? and directly return